gradle/errorprone.gradle

```diff
@@ -10,3 +10,7 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
 dependencies {
     errorprone libraries.errorprone
 }
+
+tasks.named("compileTestJava").configure {
+    options.errorprone.errorproneArgs.add("-Xep:MockitoCast:OFF")
```

Comment:
This should be removed, as this check no longer exists.

Reply:
bad merge resolve. fixed.